aws/provider.go

```diff
@@ -510,6 +510,7 @@ func Provider() *schema.Provider {
 			"aws_appconfig_configuration_profile":                     resourceAwsAppconfigConfigurationProfile(),
 			"aws_appconfig_environment":                               resourceAwsAppconfigEnvironment(),
 			"aws_appconfig_hosted_configuration_version":              resourceAwsAppconfigHostedConfigurationVersion(),
+			"aws_appconfig_deployment_strategy":                       resourceAwsAppconfigDeploymentStrategy(),
```

Comment:
Thanks so much for the rebase and updates @suzuki-shunsuke ! just a super quick note here to please move this line up above `environment` to keep the order

Reply:
Thank you for your review. Fixed 3d12fa85fcfe09ead5d7b33d1920649ef203e7c0